app/models/user.rb

```diff
@@ -203,6 +203,14 @@ def miq_groups=(groups)
     self.current_group = groups.first if current_group.nil? || !groups.include?(current_group)
   end
 
+  def change_current_group(id)
+    user_groups = miq_group_ids
+    user_groups.delete(id)
+    raise _("The user's loging group cannot be changed.") if user_groups.empty?
```

Comment:
Is there a typo in this error message?

It looks like this would be the case when the user's only group would be the one deleted.
What is a "loging" group? Even if that's supposed to be "login group" I'm still not sure what that means.